src/main/java/ChatBro/ChatBro.java

```diff
@@ -0,0 +1,33 @@
+package ChatBro;
+
+public class ChatBro {
+    static boolean isRunning;
+    public static void main(String[] args) {
+        Parser parser = new Parser();
+        TaskManager tm = new TaskManager(); // Instantiate ChatBro.TaskManager to create taskList
+        isRunning = true;
+
+        String savedTasks = Database.readFromFile();
+        String[] savedTasksSplit = savedTasks.split("\n"); // Split savedTasks by newline
+        int length = savedTasksSplit.length;
+        for (int i = 0; i < 100; i++) { // Fill taskList with null
+            tm.getList().add(null);
+        }
```

Comment:
Task list is filled with null to prevent null pointer exception with some methods, e.g. deleteTask() method. I have added a more informative comment for this method now. Thanks for the comment!